packages/react/src/themes/teams/components/TextArea/textAreaVariables.ts

```diff
@@ -1,11 +1,41 @@
-import { ResizeProperty } from 'csstype'
+import { pxToRem } from '../../../../lib'
 
 export interface TextAreaVariables {
+  backgroundColor: string
+  backgroundColorInverted: string
+  backgroundColorDisabled: string
+  colorDisabled: string
+  borderColor: string
+  borderRadius: string
+  borderWidth: string
+  fontColor: string
+  fontSize: string
+  focusBorderColor: string
```

Comment:
```suggestion
  borderColorFocus: string
```

As I remember modifier is last